src/pages/playlist/PlaylistListing.jsx

```diff
@@ -0,0 +1,57 @@
+import axios from "axios";
+import React, { useState, useEffect } from "react";
+import { useAuthContext, useStateContext } from "../../hooks";
+import { useParams } from "react-router-dom";
+import { Thumbnail } from "../../components";
+
+const PlaylistListing = () => {
+  const { state } = useStateContext();
+  const [selectedId, setSelectedId] = useState("");
+  const { playlistId } = useParams();
+  const { myToken } = useAuthContext();
+  const [isLoading, setIsLoading] = useState(true);
+  const [playlistListingData, setPlaylistListingData] = useState([]);
+  const { library } = state;
+  const { playlist } = library;
+
+  useEffect(() => {
+    (async () => {
+      try {
+        const res = await axios.get(`/api/user/playlists/${playlistId}`, {
+          headers: {
+            authorization: myToken,
+          },
+        });
+        if (res.status === 200) {

```

Comment:
auth controller in some cases returning 201 status codes also. keep that in mind next time :)

Reply:
ok